tests/parser/features/decorators/test_constant.py

```diff
@@ -1,3 +1,6 @@
+from vyper.exceptions import StructureException
```

Comment:
Out of curiousity, why `StructureException`?

Reply:
`StructureException` is what we've been using as a catch all for exceptions involving anything to do with how the contract and functions are formed.

Reply:
Makes me think of the `struct` type. Maybe something like `UsageException` or `ContextException` might be more appropiate

Reply:
@fubuloubu I'm open changing `StructureException` to something clearer but if we do I think we should change it in another PR and all at once.

Reply:
:+1: